standards/SPIFFE_Federation.md

```diff
@@ -0,0 +1,316 @@
+* tampering with the endpoint URL, especially when used in conjunction with the `https_web` profile allows an attacker to issue fraudulent keys and impersonate any identity in the corresponding trust domain
+* tampering with the endpoint profile can alter the security guarantees of the federation, for example substituting `https_spiffe` with `https_web`. If your threat model includes compromise of Web PKI (please additionally see the [Network Traffic Interception](#752-network-traffic-interception) section below), this could be considered a significant downgrade in security posture
+
+Therefore, control plane administrators must take care to source these parameters securely and input them securely. Bundle endpoint configurations can be sourced using a wide variety of methods, including, but not limited to, email, an HTTPS-protected website, a company-internal wiki, etc. Regardless of the specific method used for initial distribution of the endpoint configuration, the distribution method needs to be resistant to in-flight tampering, unauthorized modification at-rest, and malicious impersonation. For example, email is generally not resilient to tampering nor to impersonation (i.e. "spoofed" email).
+
+### 7.2. Explicitly-defined Endpoint Parameters
+Each SPIFFE Federation relationship is configured with the following parameters at minimum:
+* Trust domain name
+* Endpoint URL
+* Endpoint Profile
+
+It is important that these three parameters are configured explicitly, the values cannot be securely inferred from each other.
+
+For example, one might be tempted to infer the SPIFFE trust domain name from the host portion of the Endpoint URL. Firstly, we emphasize that a SPIFFE trust domain name need not be a registered DNS name.  Secondly, inferring this as a default is dangerous because it would allow anyone who can get a file served from a particular DNS name to assert trust roots for the SPIFFE trust domain of the same name. 
+
+Imagine a web hosting company called `spiffyhost.com`, that allows a customer, Alice, to serve web content at URLs like `https://spiffyhost.com/alice/<filename>`, and further, that `spiffyhost.com` operates an API secured by SPIFFE Federation with the SPIFFE trust domain name `spiffyhost.com`.  Imagine Alice sets up SPIFFE Federation with Bob, who is also a customer of `spiffyhost.com`, and Alice chooses to serve her trust bundle from `https://spiffyhost.com/alice/spiffe-bundle`.
+
+![A diagram illustrating the relationships between Alice, Bob, and `spiffyhost.com`](https://raw.githubusercontent.com/evan2645/spiffe/2c6d5bd6c9b7e8aafc01ec577e7be53242e18e06/standards/img/spiffe_federation_spiffyhost_example.png)  
+*Figure 5: A diagram illustrating the relationships between Alice, Bob, and `spiffyhost.com`.*
+
+If Bob’s control plane implicitly gets the trust domain name from the URL, this would then allow Alice to impersonate the trust domain spiffyhost.com!
```

Comment:
nit: "spiffyhost.com" should be written `spiffyhost.com` as it is in the previous paragraph.